tests/utils/test_numeric.py

```diff
@@ -138,3 +139,29 @@ def test_running_moments():
         true_variances = [np.var(vv) for vv in values]
         assert np.allclose(means, true_means)
         assert np.allclose(variances, true_variances)
+
+
+@pytest.mark.parametrize("min_elements", [1, 2])
+@pytest.mark.parametrize("elements_per_group", [10])
+@pytest.mark.parametrize("num_groups", [3])
+@pytest.mark.parametrize("check_num_samples", [10])
```

Comment:
Why three parametrizations instead of a single call? You are forming the cartesian product of singletons...